Gordon360/ApiControllers/HousingController.cs

```diff
@@ -69,15 +61,18 @@ public IHttpActionResult SaveApplication(string username, string [] applicants)
                 }
                 throw new BadInputException() { ExceptionMessage = errors };
             }
-            SessionViewModel currentSessionViewModel = null;
-            string sessionId = currentSessionViewModel.SessionCode;
-            string[] appIds = new string[applicants.Length];
-            for(int i = 0; i <= applicants.Length; i++){
-                appIds[i] = _accountService.GetAccountByUsername(username).GordonID;
+
+            int apartAppId = apartmentAppDetails.AprtAppID; // Set the apartAppId to -1 to indicate that an application ID was not passed by the frontend
```

Comment:
The comment would be less confusing by saying 'The apartAppId would be set to -1' or something along the line of this. The current comment sounds as if this line sets the variable to -1.

Reply:
It says that because originally the line was "int apartAppId = -1;", when I was planning to pass the applicationID in the route itself, rather than via the view model.

Reply:
We should find out if it is possible for the frontend to pass the ViewModel with AprtAppID set to 'null', since it would be more robust (and easier) to indicate "no ID number" using a 'null' rather than -1.
The value of -1 works in this case because the ID numbers can only ever be positive integers, as far as I know.

Reply:
Upon testing, it seems the C# does not allow an "int" to be set to null, so I will keep using -1 for this case

Reply:
I see. The code seems pretty clear by itself that it may not need the comment.

Reply:
This comment hasbeen improved in #446 , but it may be removed if you think it is no longer necessary

Reply:
Okay, I see it in #446, looks good